### PR TITLE
feat(properties-panel): use improved IO Mapping GUI component

### LINKED ISSUE #265: Migrate to new I/O parameter mapping GUI component
__Is your feature request related to a problem? Please describe.__

As part of the introduction we will do changes in the Input/Output tab of callActivities (see #252). Before doing this, we shall include the most-current / improved I/O mapping GUI components (see https://github.com/bpmn-io/bpmn-js-properties-panel/issues/350) so that we don't work on legacy code.

__Describe the solution you'd like__

Incorporate the new I/O GUI component (see https://github.com/bpmn-io/bpmn-js-properties-panel/issues/350) into Zeebe-Modeler.

__Describe alternatives you've considered__

Alternatively we could work with the legacy GUI component and invest in it. (This might save some effort, but (I) the outcome would not be as user-friendly, (II) Zeebe Modeler and Camunda Modeler would be out of sync design-wise and (III) we would invest into two seperate I/O mapping components.)

__Additional context__

Child of https://github.com/zeebe-io/zeebe-modeler/issues/252

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

Closes #265

__Acceptance Criteria__

* [ ] Corresponds to the concept: See #265 
* [ ] Corresponds to the design:  See #265

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Review by barmac — APPROVED** [review]
LGTM

File: client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js [hunk 1]
```diff
@@ -132,4 +135,27 @@ export function areOutputParametersSupported(element) {
...
 
+/**
+ * Get getter function for IOMapping parameters according to provided property name
+ *
+ * @param {string} property
+ *
+ * @returns {Function} Getter function for the IOMapping parameters according to provided property name
+ */
+export function determineParamGetFunc(property) {
+  if (property == 'inputParameters') {
```
Comment (barmac) [inline client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js:154]: I'd rather not use `==` comparison anywhere.

```suggestion
  if (property === 'inputParameters') {
```

**Comment by pinussilvestrus** [comment]
I planned to have a deeper look into this for a review tomorrow. If you merged it until than, I'd also be fine :-)

File: client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js (hunk 1 above)
Comment (MaxTru) [reply client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js:154]: Done

**Pull request merged**